tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.7.0, tc, 13096
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea
+
+1. Open Chrome browser.
```

Comment:
Ohh I see your comment above. Still getting used to github ;)
I'll make change after weekend